src/components/ui/close-button.jsx

```diff
@@ -0,0 +1,20 @@
+function _nullishCoalesce(lhs, rhsFn) {
+  if (lhs != null) {
+    return lhs
+  } else {
+    return rhsFn()
+  }
+}
+import { IconButton as ChakraIconButton } from '@chakra-ui/react'
+import * as React from 'react'
+import { LuX } from 'react-icons/lu'
+
+export const CloseButton = React.forwardRef(function CloseButton(props, ref) {
+  return (
+    <ChakraIconButton variant='ghost' aria-label='Close' ref={ref} {...props}>
+      {_nullishCoalesce(props.children, () => (
+        <LuX />
+      ))}
```

Comment:
Kind of curious, what's up with the _nullishCoalesce function? I see what it is doing, I'm just making sure you know 😄 .